modules/core/src/test/java/org/apache/fluo/core/data/BytesBuilderTest.java

```diff
@@ -71,6 +71,17 @@ public void testSingleByte() {
   }
 
   @Test
+  public void testArraySection() {
```

Comment:
The following additional test would be nice :
 * calling append w/ a non-zero offset
 * calling this new append method multiple times on a single BytesBuilder